### PR TITLE
Deprecate-vue-cli-service

### LINKED ISSUE #219: [CHORE]: Deprecate vue-cli-service ESLint & use base ESLint.
**Is your feature request related to a problem? Please describe.**
This project was originally built with Vue CLI 3 and has transitioned into a monorepo with lerna and yarn workspaces.

The challenge is that different packages end up having different ESLint configs. I'd like for these to be unified into one so contributors are able to contribute comfortable without insecurities about linting.

**Describe the solution you'd like**
1. Figure out how to setup ESLint config that we can execute from the root in specific/all packages.
2. Extend [`@nuxt/eslint-config`](https://github.com/nuxt/eslint-config). I feel this config fits nicely between pure Javascript modules and Vue components.

### PR DESCRIPTION
**Pull request by Easybuoy**
Closes #219 
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ sw.js
 **/lib/plugin.js
 **/.nuxt
 **/coverage
-**/packages/chakra-ui-theme/src/*
\ No newline at end of file
+**/packages/chakra-ui-theme/src/*
```
Comment (codebender828) [inline .eslintignore:8]: I think this `packages/chakra-ui-theme` package should also be linted also. We had previously excluded it for some reason, but I cannot recall.

File: package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "build:dev": "yarn workspace @chakra-ui/vue run rollup --config rollup.dev.config.js",
     "build": "yarn workspace @chakra-ui/vue build",
     "dev": "yarn workspace @chakra-ui/vue dev",
-    "lint": "vue-cli-service lint packages/chakra-ui-core", 
+    "lint": "eslint packages",
```
Comment (codebender828) [inline package.json:26]: 👍

**Comment by codebender828** [comment]
In the `.github/workflow.yml` file, you could also include:
```yml
- run: yarn lint
```

This will enforce linting for pull requests before they are merged. Thanks!

File: .eslintignore (hunk 1 above)
Comment (Easybuoy) [reply .eslintignore:8]: Okay, I'd remove it so it can be listed as well.

**Comment by Easybuoy** [comment]
Added the requested changes @codebender828

**Comment by codebender828** [comment]
@all-contributors please add @Easybuoy for tool

**Pull request merged**